source/_integrations/apcupsd.markdown

```diff
@@ -130,6 +130,10 @@ Some sensors are disabled by default, since they provide information that is onl
 - UPS Transfer from Battery (XOFFBATT)
 - UPS Transfer to Battery (XONBATT)
 
+## Data updates
+
+The integration will fetch data from your APC UPS Daemon every 60 seconds by default.
```

Comment:
```suggestion
The integration {% term polling polls %} data from your APC UPS Daemon every 60 seconds by default.
```
we have a glossary ref for that :smile:

Reply:
@c0ffeeca7 Niiice! Should we then also update the template? I was mostly following the template and thought perhaps "fetch" and "push" are the terms we recommend: https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown#data-updates 😃

Reply:
oh, yes! Thank you for the catch, @yuxincs :raised_hands: https://github.com/home-assistant/home-assistant.io/pull/38797